src/eos-dropbox-app

```diff
@@ -47,6 +48,12 @@ def get_default_dropbox_directory():
     if os.path.isdir(default_dir):
         return default_dir
 
+    # No 'Dropbox' directory found, our last attempt will be to look for a Dropbox
+    # team folder, used in 'business' accounts (e.g. 'Dropbox (Endless Team)')
+    team_dir = glob.glob(default_dir + ' (*)')
+    if team_dir:
+        return team_dir[0]
+
```

Comment:
It's a fallback, and even if the use case is not very likely to happen, it's still possible (e.g. people migrating Dropbox dirs from another computer before configuring it locally, people removing -accidentally or not- the `.dropbox` directory...).

I think it's harmless in most cases and useful in some, so I'd rather keep it just in case